src/Rector/Contrib/PHPUnit/SpecificMethodRector.php

```diff
@@ -104,16 +103,40 @@ private function renameMethod(MethodCall $methodCallNode): void
         [$trueMethodName, $falseMethodName] = $this->oldToNewMethods[$this->activeFuncCallName];
 
         if ($oldMethodName === 'assertTrue' && $trueMethodName) {
-            $methodCallNode->name = $trueMethodName;
+            /** @var string $trueMethodName */
+            $methodCallNode->name = new Identifier($trueMethodName);
         } elseif ($oldMethodName === 'assertFalse' && $falseMethodName) {
-            $methodCallNode->name = $falseMethodName;
+            /** @var string $falseMethodName */
+            $methodCallNode->name = new Identifier($falseMethodName);
         }
     }
 
     private function moveArgumentUp(MethodCall $methodCallNode): void
     {
-        /** @var FuncCall $funcCall */
-        $funcCall = $methodCallNode->args[0]->value;
-        $methodCallNode->args[0] = $funcCall->args[0];
+        $funcCallOrEmptyNode = $methodCallNode->args[0]->value;
+        if ($funcCallOrEmptyNode instanceof FuncCall) {
+            $methodCallNode->args[0] = $funcCallOrEmptyNode->args[0];
+        }
+
+        if ($funcCallOrEmptyNode instanceof Empty_) {
+            // what should be here on the right part instead to fix this? :)
+            $methodCallNode->args[0] = $funcCallOrEmptyNode->args[0];
```

Comment:
@carusogabriel I got question for you. Try to figure this out :)

Now it fails, because we try to get args from `Empty_` node. The [`Empty_` node class might help](https://github.com/nikic/PHP-Parser/blob/master/lib/PhpParser/Node/Expr/Empty_.php)

```
1) Rector\Tests\Rector\Contrib\PHPUnit\SpecificMethodRector\Test::test
Undefined property: PhpParser\Node\Expr\Empty_::$args

/var/www/Rector/src/Rector/Contrib/PHPUnit/SpecificMethodRector.php:123
```

Reply:
Okay, let me see. We get `args`, that contains what in inside the function, right?

So we need to find a method that returns what in inside `empty`? :man_technologist:

Reply:
Find it! We call `$funcCallOrEmptyNode->expr`, with is the expression passed to `empty()` :smile:

Reply:
I gonna try to work on `array_key_exists` fix now.

Reply:
@carusogabriel That is...


... correct :+1:  Wow, you're getting better than me faster than I thought :1st_place_medal: 


I'll finish this `empty()` issue now and merge this PR.

Feel free to send PR for the `array_key_exists` yourself.